wire-ios-transport/Source/Public/ZMUpdateEvent.swift

```diff
@@ -115,7 +115,7 @@ extension ZMUpdateEventType {
         case .conversationRename:
             return "conversation.rename"
         case .conversationProtocolUpdate:
-            return "conversation.protocol-update"
+            return "conversation.protocol-update" // TODO: this is the event to be used
```

Comment:
What does this TODO mean? 
Can it be fixed before merging?

Reply:
Sorry removed it